operator.go

```diff
@@ -0,0 +1,149 @@
+package cloudconfig
+
+import (
+	"bytes"
+	"html/template"
+	"strings"
+
+	"github.com/giantswarm/awstpr"
+)
+
+var (
+	awsFilesMeta []FileMetadata = []FileMetadata{
+		FileMetadata{
+			AssetPath:   "templates/operator/aws/decrypt-tls-assets",
+			Path:        "/opt/bin/decrypt-tls-assets",
+			Owner:       "root:root",
+			Permissions: 0700,
+		},
+	}
+	awsUnitsMeta []UnitMetadata = []UnitMetadata{
+		UnitMetadata{
+			AssetPath: "templates/operator/aws/decrypt-tls-assets.service",
+			Name:      "decrypt-tls-certs.service",
+			Enable:    true,
+			Command:   "start",
+		},
+	}
+	kvmUnitsMeta []UnitMetadata = []UnitMetadata{
+		UnitMetadata{
+			AssetPath: "templates/operator/kvm/add-route-bridge.service",
+			Name:      "add-route-bridge.service",
+			Enable:    true,
+			Command:   "start",
+		},
+	}
+)
+
+type FileMetadata struct {
+	AssetPath   string
+	Path        string
+	Owner       string
+	Permissions int
+}
+
+type FileAsset struct {
+	Metadata FileMetadata
+	Content  []string
+}
+
+type UnitMetadata struct {
+	AssetPath string
+	Name      string
+	Enable    bool
+	Command   string
+}
+
+type UnitAsset struct {
+	Metadata UnitMetadata
+	Content  []string
+}
+
+type OperatorExtension interface {
+	GetFiles() ([]FileAsset, error)
+	GetUnits() ([]UnitAsset, error)
+}
```

Comment:
I would really like to fix the naming if there is no better reason to keep it as it is. See

> I would like to align the naming to the golang convention. Getters do not have the get prefix. See https://golang.org/doc/effective_go.html#Getters.